build.js

```diff
@@ -23,14 +25,28 @@ fetch(endpoint)
 	 */
 	sorted.forEach((currency, index) => {
 		const {Name, CoinName, ImageUrl} = currency;
-		symbols[Name] = CoinName;
+		symbols[Name] = {
+			name: CoinName
+		};
 
 		// download the image for future use
 		if (ImageUrl) {
+			symbols[Name].imageUrl = `${ghBaseUrl}/${Name}.png`;
+
 			spinner.text = `${chalk.gray(index)} ${Name}`;
 			spinner.render();
+
+			const extension = ImageUrl.split('.').pop();
+			const ImageFile = `${Name}.${extension}`;
+			const ImagePath = path.join('images', ImageFile);
+
+			if (fs.existsSync(ImagePath)) {
```

Comment:
maybe we need to add a --force flag to the build then, because this checks for an existing image, but sometimes the icons change, and we will want a way to grab the updates images.

Reply:
Yes that's what I was thinking, a force flag should be beneficial.